README.md

```diff
@@ -64,7 +65,13 @@
     * [7.4.8. Using Reliable Topic](#748-using-reliable-topic) 
     * [7.4.9. Using PN Counter](#7412-using-pn-counter)
     * [7.4.10. Using Flake ID Generator](#7413-using-flake-id-generator)
-    * [7.4.11. Using Transactions](#7414-using-transactions)
+    * [7.4.11. CP Subsystem](#7411-cp-subsystem)
```

Comment:
Isn't it better, for the sake of consistency, that we use structures names as Semaphore, Latch (or CountDown Latch) and Atomic Reference?

Reply:
Changed with what is used at the API for consistency of CP structures.